unit/config.json

```diff
@@ -0,0 +1,28 @@
+{
+	"listeners": {
+		"*:80": {
+			"pass": "routes"
+		}
+	},
+	"routes": [],
+	"applications": {},
+	"settings": {
+		"http": {
+			"header_read_timeout": 10,
+			"body_read_timeout": 10,
+			"send_timeout": 10,
+			"idle_timeout": 10,
+			"max_body_size": 3146000,
```

Comment:
This is in bytes ? If it is isnt 3mb way small ? (thinking about images). Or is this the request body ?

Reply:
yes, bytes. I lowered the default 8MB value because of the small server. But we can make it higher if needed.

Reply:
It is that I really like images and memes and some of the memes are gifs =P. But we dont have much stuff for now